tests/virt_autotest/update_package.pm

```diff
@@ -58,6 +58,9 @@ sub run {
     # turn on debug for libvirtd & enable journal with previous reboot
     enable_debug_logging if is_x86_64;
 
+    # detect coredump or other failures
+    check_host();
```

Comment:
Did you verify this on other archs ?

Reply:
Not yet. @waynechen55 @guoxuguang Could you please review my PR and verify on arm and s390?

Reply:
sure, got it. quick question,  why do we need `check_host()` at update_package test module?  means that detect kernel core dump during update packages on our SLE VM host, right?

Reply:
I mean to. As for my own, I like more checks to narrow down where the coredump happens, but I know it will be wordy. Welcome team discussion on how and where we handle coredump.

Reply:
IMHO. We should react to a pattern instead of to a single case unless the case indicates an error or being insufficient. I thing bring up a interface that leads to host crash at a time if an isolated single case, I do not think you should introduce some large-scale changes. You can keep your new functions, but do not apply it everywhere IMHO.

Reply:
two failures here, coredump and crash. bringing up an interface just result in guest coredump rather than crash. Host coredump happens after host bootup, so the check_coredump in login_console() is more necessary than in update_package, except some steps in update_package triggers the coredump.
check_host() can be removed in update_package() if you think it is noisy.

Reply:
Yes, I think you should remove check_host from update_package.pm.